lfroms/nimbus
# Convert database to PostGIS

### What is the problem being solved in this PR?
This PR modifies the database so that it uses PostGIS as opposed to plain PostgreSQL. This will allow the service to perform geospatial operations in order to determine country boundaries, for example.

### Related issues or PRs
N/A

### How is this accomplished and why did you choose this approach?
By adding the `activerecord-postgis-adapter` gem and adjusting adapter settings.

### How to test changes
1. Run `rails db:gis:setup`.
2. Ensure that the app still works.

### Checklist
